httpobs/database/database.py

```diff
@@ -140,15 +140,17 @@ def insert_test_results(site_id: int, scan_id: int, tests: list, response_header
                         (site_id, scan_id, name, expectation, result, passed, dumps(test), score_modifier))
 
         # Now we need to update the scans table
-        score, grade = get_grade_for_score(score)
+        score, grade, likelihood_indicator = get_grade_and_likelihood_indicator_for_score(score)
 
         # Update the scans table
         cur.execute("""UPDATE scans
-                         SET (end_time, tests_failed, tests_passed, grade, score, state, response_headers) =
+                         SET (end_time, tests_failed, tests_passed, grade, score, likelihood_indicator,
```

Comment:
likewise, this value is not returning risk - it is unable to return risk as there is no impact component and no context around the scored values. the likelihood indicator is used by the servicemap among other indicators to calculate the risk that is represented in our heatmap